core/templates/pages/moderator-page/moderator-page.component.spec.ts

```diff
@@ -255,3 +255,16 @@ describe('Moderator Page Component', () => {
     );
   });
 });
+
+it('should display an error message for invalid exploration IDs', fakeAsync(() => {
```

Comment:
Why is this outside the scope of the `describe` block and not formatted by prettier?